migrations/2016_06_27_000000_create_mediable_tables.php

```diff
@@ -17,18 +18,16 @@ public function up()
             Schema::create(
                 'media',
                 function (Blueprint $table) {
-                    $table->increments('id');
+                    $table->id();
                     $table->string('disk', 32);
                     $table->string('directory');
                     $table->string('filename');
                     $table->string('extension', 32);
                     $table->string('mime_type', 128);
-                    $table->string('aggregate_type', 32);
-                    $table->integer('size')->unsigned();
+                    $table->string('aggregate_type', 32)->index();
+                    $table->unsignedInteger('size');
```

Comment:
`unsignedInteger` has been deprecated in Laravel 

https://laravel.com/docs/11.x/upgrade#floating-point-types

<img width="678" alt="image" src="https://github.com/plank/laravel-mediable/assets/7753600/aa0686e3-1873-446f-aaf1-2b7a30c080c3">

Reply:
That column isn't a float.